enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -10,7 +10,8 @@
             [metabase.server.middleware.session :as mw.session]
             [metabase.server.request.util :as request.u]
             [metabase.util.i18n :refer [trs tru]]
-            [ring.util.response :as resp])
+            [ring.util.response :as resp]
+            [clojure.string :as str])
```

Comment:
you'll need to sort your ns to satisfy the namespace linter which is failing the build right now.

Reply:
Oops. Yea, thanks.